roles/ansible_service_broker/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 # do any asserts here
 
 - include: install.yml
-  when: ansible_service_broker_install | default(false) | bool
+  when: ansible_service_broker_install | default(true) | bool
 
```

Comment:
Maybe we should add `- not ansible_service_broker_remove | default(false) | bool`?

Reply:
:+1: not sure how often someone would set both to true but sounds good to me..

though, what would be the priority if both `ansible_service_broker_install=True` and `ansible_service_broker_remove=True`?